reana_commons/utils.py

```diff
@@ -181,9 +181,13 @@ def build_caching_info_message(job_spec,
     return caching_info_message
 
 
-def get_workspace_disk_usage(workspace, summarize=False):
+def get_workspace_disk_usage(workspace, summarize=False, block_size=None):
     """Retrieve disk usage information of a workspace."""
-    command = ['du', '-h']
+    command = ['du']
+    if block_size:
+        command.append('-{}'.format(block_size))
+    else:
+        command.append('-h')
     if summarize:
         command.append('-s')
     else:
```

Comment:
Please inclease reana-commons YYYMMDD version at the same time, so that we can push this to PyPI later